api/src/utils/query-utils.js

```diff
@@ -121,6 +121,15 @@ exports.isRecordAnonymous = function(record) {
     return false;
   }
 
+  if (record.issuedTo.forceAnonymous) {
+    // record manually set to anonymous
+
+    // Note: this value only indicates that the user has toggled this record to be anonymous.  If it is set to null or
+    //   false, that only tells us the user hasn't toggled this value, and does not necessarily mean this record is
+    //   public (not anonymous).
```

Comment:
Alignment of comment text.

Additionally I think the function name should be something like "isConsideredAnonymous", because we may want to have different rules around what defines anonymous based on particular record types.  It's possible LNG, NRCED, and the incoming 3rd flavour may want to treat things a bit differently - either by record type, or by business area.

Reply:
Yeah I intentionally indented the comment a bit so it would stand out as a paragraph, rather than separate lines. But I'm not worried about the format, so I can change it.

Yeah I can change the function name, sounds good.